api/libretime_api/schedule/models/show.py

```diff
@@ -69,6 +69,28 @@ def live_enabled(self) -> bool:
     auto_playlist_enabled = models.BooleanField(db_column="has_autoplaylist")
     auto_playlist_repeat = models.BooleanField(db_column="autoplaylist_repeat")
 
+    intro_playlist = models.ForeignKey(
+        "schedule.Playlist",
+        on_delete=models.DO_NOTHING,
+        blank=True,
+        null=True,
+        db_column="intro_playlist_id",
+        related_name="intro_playlist",
+    )
+
+    override_intro_playlist = models.BooleanField(db_column="override_intro_playlist")
```

Comment:
Why use a boolean field AND a nullable ID? Can't we consider a null ID as "disabled" value?

Reply:
This comes from the UI logic in the show builder. the has_autoplaylist works the same way (and hides parts of the UI when unchecked). I will change though, since it saves 2 clicks when editing.